README.md

```diff
@@ -1,15 +1,18 @@
-Follow these commands:
+# Follow these commands
 
+```
 Py -3 -m venv venv
 
 venv\Scripts\activate
 
 pip install flask
-
+```
 For the running of the app, do:
 
+```
 set FLASK_APP=app.py
 
 set FLASK_ENV=development
 
 flask run
+```
```

Comment:
Please use the readme [here](https://github.com/mentorpaired/core#readme) as a guide to making this nicer